app/src/main/AndroidManifest.xml

```diff
@@ -9,7 +9,8 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity">
+        <activity android:name=".MainActivity"
+            android:exported="true">
```

Comment:
Is this necessary?

Reply:
Yes, apps targeting android 12 and above must declare this in activity manifest